.circleci/config.yml

```
@@ -0,0 +1,38 @@
+version: 2.1
+orbs:
+  architect: giantswarm/architect@0.2.0
```

```suggestion
  architect: giantswarm/architect@0.4.3
```
https://github.com/giantswarm/giantswarm/blob/master/processes/appcatalog.md#packaging-and-pushing-apps-into-an-app-catalog